argonaut/src/main/scala/io/github/mcgizzle/circe/package.scala

```diff
@@ -0,0 +1,3 @@
+package io.github.mcgizzle
+
+package object circe extends ArgonautDecoder
```

Comment:
shouldn't this package be called argonaut?

Reply:
aghh intelliJ rename was doing untold things